server/views/components/assessment/_assessment-area.njk

```diff
@@ -64,12 +64,14 @@
                         {% if assessment.criminogenicNeedsScore %}
                             <div class="assessment-score">
                                 <h3 class="govuk-heading-s">{{ assessment.title }} need score</h3>
-                                <p class="govuk-body">{{ assessment.criminogenicNeedsScore }} out of {{ assessment.upperBound }}. (Scores above 3 are high-scoring.)</p>
+                                <p class="govuk-body">{{ assessment.criminogenicNeedsScore }} out of {{ assessment.upperBound }}. (Scores above
+                                    {{ assessment.thresholdValue }} are high-scoring.)</p>
```

Comment:
I only just realised that this isn't use the locale files... oops. We'll resolve that in a follow up to re-work/clean-up a bunch of this page.

Reply:
yes, there's loads of localisation needed in this page!